src/app/components/Text/index.stories.jsx

```diff
@@ -11,9 +11,9 @@ const paragraphBlock = (blockId, text) => ({
 
 const props = {
   blocks: [
-    paragraphBlock('01', 'This is a paragraph.'),
-    paragraphBlock('02', 'This is another paragraph.'),
-    paragraphBlock('03', 'This is a final paragraph.'),
+    paragraphBlock('01', 'It was designed by London-based florist Philippa Craddock, who also created the floral displays for St George\'s Chapel and St George\'s Hall using locally sourced foliage, [which were later donated to local hospices](/news/articles/c000000000ro).'),
```

Comment:
Duplication between here and index.test.jsx

Reply:
The duplication will exist until we can look at [fixture data](https://github.com/bbc/simorgh/issues/243).